processing_r/processing/algorithm.py

```diff
@@ -218,7 +218,10 @@ def process_metadata_line(self, line):
                                                                     self.tr('HTML files (*.html)'), optional=True))
             return
         if line.lower().strip().startswith('dontuserasterpackage'):
-            self.use_raster_package = False
+            self.r_templates.use_raster = False
+            return
+        if line.lower().strip().startswith('dontusesfpackage'):
```

Comment:
What do you think about replacing `dontuserasterpackage` and `dontusesfpackage` with something a little easier to read and understand the implications of -- e.g. `use_gdal_for_raster_imports` and `use_gdal_for_vector_imports`?

Reply:
Sounds like a good idea. The current strings are way to unintelligible.

But there is a lot ambiguity for R users in using metadata names you suggested. GDAL is used to read data regardless which package you actually use. Both `rgdal` and `sp`, which are older, and `sf`, which is the new, are based on GDAL. It might be better to refer directly to package `rgdal` which outputs spatial classes from `sp` package. 

So how about `load_vector_using_rgdal` and `load_raster_using_rgdal`?

Reply:
Perfect!

Reply:
Done. 

Thinking about it, it seems that changing the names of `showplots` and `passfilenames` might make sense too. Either just use the underscore variants `show_plots` and `pass_filenames`. 

Maybe even change `showplots` to `output_plots_to_html` or something like that. To make it clear that there is special type of output with plots. The `showplots`  keyword seems like after the execution of the script a window will open and show user the plots, which is not exactly the case. It is just a suggestion though.

Reply:
Yes, +1 for output_plots_to_html over show plots! And also +1 to standardising with underscores. We just need to keep checks for the old versions too, so we don't break existing scripts

Reply:
Done.